promoted_python_delivery_client/client/delivery_request.py

```diff
@@ -10,4 +9,4 @@ class DeliveryRequest:
     request: Request
     experiment: Optional[CohortMembership] = None
     only_log: bool = False
-    insertion_page_type: InsertionPageType = InsertionPageType.UNPAGED
+    insertion_start: int = 0
```

Comment:
Side-comment - in promoted-ts-client, I make this a required parameter so they need to evaluate it.  I'm worried about bugs around this.

Reply:
I would lean toward not doing that from an API design standpoint since 0 the default is usually right and the parameter is backed by validation (and also all the existing parameters are optional). But I'll change it if you feel strongly about it.